capnslog/pkg_logger.go

```diff
@@ -37,13 +37,17 @@ func (p *PackageLogger) internalLog(depth int, inLevel LogLevel, entries ...inte
 	}
 }
 
+func (p *PackageLogger) internalLogf(depth int, inLevel LogLevel, format string, a ...interface{}) {
+	p.internalLog(depth, inLevel, fmt.Sprintf(format, a...))
+}
+
 func (p *PackageLogger) LevelAt(l LogLevel) bool {
 	return p.level >= l
 }
 
 // Log a formatted string at any level between ERROR and TRACE
 func (p *PackageLogger) Logf(l LogLevel, format string, args ...interface{}) {
```

Comment:
How about just having all the other functions you changed call this one. Then no need for internalLogf at all.